lib/lib_rss.php

```diff
@@ -15,6 +15,8 @@ function mb_strcut($str, $start, $length = null, $encoding = 'UTF-8') {
 	openlog('FreshRSS', LOG_CONS | LOG_ODELAY | LOG_PID, LOG_USER);
 }
 
+require_once LIB_PATH . DIRECTORY_SEPARATOR . 'autoload.php';
```

Comment:
`/` works on all platforms and is shorter :-)

Reply:
Except that if you read too fast like I did, you forget the `/` :)